src/esn.calendar.libs/app/components/time-select/time-select.controller.spec.js

```diff
@@ -0,0 +1,237 @@
+'use strict';
+
+/* global chai: false */
+/* global sinon: false */
+
+const expect = chai.expect;
+
+describe('the calTimeSelectController', function() {
+  var $controller, calMoment, $scope, testingDate;
+
+  beforeEach(function() {
+    angular.mock.module('esn.calendar.libs', function($provide) {
+      $provide.value('esnDatetimeService', () => {});
+    });
+
+    angular.mock.inject(function(_$controller_, _calMoment_, _$rootScope_) {
+      $controller = _$controller_;
+      calMoment = _calMoment_;
+      $scope = _$rootScope_.$new();
+    });
+
+    testingDate = calMoment('2020-10-02 14:00:00Z').utc();
+  });
+
+  function initCtrl(bindings) {
+    const controller = $controller('calTimeSelectController',
+      { $scope, $element: $('<div></div>') },
+      {
+        ...bindings,
+        date: testingDate
+      });
+
+    controller.$onInit();
+    $scope.$digest();
+
+    return controller;
+  }
+
+  describe('the $onInit function', function() {
+    it('should initialize the event selected time correctly in 24H format', function() {
```

Comment:
Tests description are mixed with the next one

Reply:
Ye @rezk2ll I think you should have similar labels for the tests.